torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2142,6 +2142,20 @@ def create_symintnode(self, sym: "sympy.Expr", *, hint: Optional[int]):
             return int(sym)
         return SymInt(SymNode(sym, self, int, hint))
 
+    def create_symint_and_symbol(self, value, source, dynamic_dim):
+        # TODO: This is wrong wrong wrong, create_symbol will
+        # generate something that is non-negative, but this is
+        # not a sound assumption to make.
+        # Not fixing as this was a preexisting condition.
+        return self.create_symintnode(
+            self.create_symbol(
+                value,
+                source=source,
+                dynamic_dim=dynamic_dim,
```

Comment:
was it significant to remove constraint_dim=none here?  

was there any actual change other than code motion?

Reply:
It is just code motion so it can be reused in the next PR.

constraint_dim=none is the default value.